challenge.rb

```diff
@@ -45,10 +49,17 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ac_working && current_temp.to_i > desired_temp.to_i
+    return "Turn on the AC."
+  elsif ac_working && current_temp == desired_temp
+    return "Just right!"
+  elsif !ac_working &&  current_temp > desired_temp
+    return "Fix the AC now! It's hot!"
+  else
+    "Fix the AC whenever you have the chance. It's cool."
+  end
```

Comment:
Correct! In fact, you don't need the other three `return`s either; in any of those cases, the string would be the last line of code evaluated.